challenges/2021-04-06-intervening-verbiage/solutions/beccani.java

```diff
@@ -0,0 +1,66 @@
+package io.codeconnector.codedojo;
+
+/**
+ * # Intervening Verbiage
+ *
+ * Given two words in a string of words, count the number of other words between
+ * the two words you are given.
+ *
+ * ## Business Rules/Errata
+ *
+ * - Your solution will take in three pieces of input: a string containing a
+ * space-separated list of words, and two other strings representing the first
+ * and second words, respectively.
+ * - You may ignore all punctuation and capitalization.
+ * - If your two words are not in the string of words, alert the user to an error.
+ * - You should return a count of the words between your two words as an integer,
+ * not the words themselves.
+ * - If your words appear multiple times in the list of words, count words that
+ * fall between the first  instance of each.
+ *
+ * ## Examples
+ *
+ * ```
+ * list_of_words = "There was an old lady who lived in an outhouse";
+ * count_words_between("lady", "outhouse", list_of_words)  // 4
+ * count_words_between("an", "outhouse", list_of_words)    // 6
+ * ```
+ *
+ * - The words 'who', 'lived', 'in', 'an' appear between 'lady' and 'outhouse'.
+ * - The words 'old', 'lady', 'who', 'lived', 'in', 'an' appear between the
+ * first instance of 'an' and 'outhouse'.
+ */
+public class InterveningVerbiage {
+    public int countWordsBetween(
+        String start,
+        String stop,
+        String words
+    ) throws IllegalArgumentException
+    {
+        // You should put the challenge solution here
+        String[] wordsArray = words.split(" ");
+        System.out.println(wordsArray);
+        int count = 0;
+        boolean counting = false;
+
+        for (String word : wordsArray) {
+            System.out.println(word);
+            if (word.equalsIgnoreCase(stop)){
+                return count;
+            }
+            if (counting==true) {
```

Comment:
@beccani it looks like you have the file ready to commit. Minus @drkennetz suggestion about 
```suggestion
            if (counting) {
```

Reply:
It's odd. There's one commit that appears to have the change in it, but the most recent version doesn't seem to.

Reply:
659d91a8c2880a1b152a61c14cb0d221b32e5881 appears to have it.

Reply:
(Note, here's an interesting thing about gihub)

It allows me to make a commit-able suggesting.  Feel free to NOT commit my suggestion 😄  this was just a fun way to display a suggestion diff


Nice work on all of this

Reply:
I think there was a rebase that removed it?

Reply:
https://github.com/codeconnector/CodingDojo/pull/21/commits/0037dfad28a565332db7c73e6288ad08be4e8c2d

Reply:
Nah, that was me. I replaced the contents of InterveningVerbiage.java with the original contents to remove it from the commit. (There was some merge cruft in there as well). Shouldn't have touched the beccani.java file.